gcovr/gcov.py

```diff
@@ -604,6 +605,73 @@ def check_unrecognized_lines(self, ignore_parse_errors):
         sys.exit(1)
 
 
+class gcovr_executer:
```

Comment:
Other classes use CamelCase.
```suggestion
class GcovrExecuter:
```

Reply:
This class will be removed, see next comment.